app/src/main/java/org/gnucash/android/ui/settings/BackupPreferenceFragment.java

```diff
@@ -422,12 +437,32 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
 			case REQUEST_RESOLVE_CONNECTION:
 				if (resultCode == Activity.RESULT_OK) {
 					mGoogleApiClient.connect();
-					Preference pref = findPreference(getString(R.string.key_dropbox_sync));
+					Preference pref = findPreference(getString(R.string.key_google_drive_sync));
 					if (pref == null) //if we are in a preference header fragment, this may return null
 						break;
-					toggleDropboxPreference(pref);
+					toggleGoogleDrivePreference(pref);
 				}
 				break;
 		}
 	}
+
+	@Override
+	public void onGoogleDriveFolderCreated(String folderId) {
```

Comment:
The parameter isn't used now, but the listener interface provides it for those listener(s) that would need it. Other examples of such methods are ListView adapter events that you don't always use all the parameters.